docs/custom_quantum_functions.md

```diff
@@ -90,39 +90,55 @@ keyrecord_t record {
 
 # LED Control
 
-This allows you to control the 5 LED's defined as part of the USB Keyboard spec. It will be called when the state of one of those 5 LEDs changes.
+The firmware allows to read the 5 LED's defined as part of the USB Keyboard spec:
 
 * `USB_LED_NUM_LOCK`
 * `USB_LED_CAPS_LOCK`
 * `USB_LED_SCROLL_LOCK`
 * `USB_LED_COMPOSE`
 * `USB_LED_KANA`
 
+These five constants corresponds to the positional bits of the host LED state.
+There are two ways to get the host LED state:
+
+* by implementing `led_set_user()`
+* by calling `host_keyboard_leds()`
+
+## `led_set_user()`
+
+This function will be called when the state of one of those 5 LEDs changes.
+It receives the LED state as parameter.
+Use the `IS_LED_ON(USB_LED, LED_NAME)` and `IS_LED_OFF(USB_LED, LED_NAME)`
+macros to check the LED status.
+
+Note that `host_keyboard_leds()` may reflect the new value already
```

Comment:
This is awkwardly worded, and inappropriately formatted. 

```suggestion
!> `host_keyboard_leds()` may return a newer value than `led_state_set()` has.
```

Reply:
“_before_” was missing in the sentence indeed. I have reworded it a bit, merged the two lines and replaced the "Note that" with `!>` as suggested.